lib/tic_tac_toe_app.rb

```diff
@@ -1,26 +1,48 @@
 require 'rack'
+require 'controller'
 require 'display'
 require 'request_handler'
 
 class TicTacToeApp
+
+  def initialize
+    @controller = Controller.new
+  end
+
   def rack_up
+    controller = @controller
+
     Rack::Builder.new do |env|
-      map "/" do
+
+      map "/start" do
         run (Proc.new do |env|
-          [200, {}, [""]]
+          request = RequestHandler.new(Rack::Request.new(env))
+          show = Display.new.home_page
+          if request.start == "start"
+            ['302', { 'Content-Type' => 'text/html', 'Location' => "/play" }, []]
+          else 
+            show = Display.new.home_page
+            [200, {}, [show]]
+          end
         end)
       end
 
       map "/play" do
         run (Proc.new do |env|
           request = RequestHandler.new(Rack::Request.new(env))
-          current_board = request.board 
-          display_board = request.mark_board(current_board)
-          show = Display.new.generate_display(display_board)
-          [200, {}, [show]]
+          player_move = request.player_move
+          if player_move != 0
```

Comment:
the magic number here seems a bit arbitrary/a bit of a hack. It's not clear why this is necessary without knowing the context. If this is for protecting the case where the user has yet to make a move, I'd try and make that a bit more explicit.  Something like.. `if move_made?(request)...`

Again, even extracting as a method could provide some more clarity.